sql/src/test/java/org/apache/druid/sql/calcite/schema/SystemSchemaTest.java

```diff
@@ -381,6 +381,8 @@ public void setUp(@TempDir File tmpDir) throws Exception
 
   private final DateTime startTime = DateTimes.nowUtc();
 
+  private final String version = new DruidNode("s1", "localhost", false, 8081, null, true, false).getVersion();
```

Comment:
I don't think `DruidNode` exposes any `getVersion` method.
Also, if we just need the version, why instantiate a `DruidNode`?